core/src/main/java/org/wildfly/extras/sunstone/api/impl/Config.java

```diff
@@ -253,17 +260,17 @@
             public static final String WAIT_FOR_PORTS = "ec2.waitForPorts";
             /**
              * Unencoded Value of <a href="http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/user-data.html">EC2 User data</a>.
-             * This property takes precedence over the {@value #USER_DATA_FILE}.
+             * This property takes precedence over the {@value #BOOT_SCRIPT_FILE}.
              *
-             * @see #USER_DATA_FILE
+             * @see #BOOT_SCRIPT_FILE
              */
             public static final String USER_DATA = "ec2.userData";
             /**
              * Path to a file with unencoded value of
              * <a href="http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/user-data.html">EC2 User data</a>. This property is
-             * only used when {@value #USER_DATA} property is not specified.
+             * only used when {@value #BOOT_SCRIPT} property is not specified.
              *
-             * @see #USER_DATA
+             * @see #BOOT_SCRIPT
              */
```

Comment:
I think it'll be better to just remove the javadoc for EC2 properties. It's a legacy thing anyway and everything should be in `ec2-README.md`. I'l put it in my TODO list and create a PR.